### DISCUSSION TITLE
Shortcut to set heading in markdown cells

### DISCUSSION THREAD

**Discussion by RafalSkolasinski** [microsoft/vscode-jupyter#10769] [New Ideas]
In normal Jupyter Notebook or Jupyter Lab one can hit 
```
<ESC>; <number = 1, 2, 3, 4, etcr>
```
to set e.g. top level or second level heading. 

This seems to be missing from the VS Code when working with notebooks. It is a major impact to my flow :(

**Comment by rchiodo** [comment]
Thanks. Sounds like a good idea. We should add it to the jupyter keymaps extension.

**Comment by jsulopzs** [comment]
Any updates on this?

**Comment by jankaWIS** [comment]
Same question, I'd also really appreciate that feature. Maybe this time there will be higher interest:
https://github.com/microsoft/vscode-jupyter/issues/10780
